product_docs/docs/pgd/5/rel_notes/pgd_5.5.0_rel_notes.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "EDB Postgres Distributed 5.5.0 release notes"
+navTitle: "Version 5.5.0"
+---
+
+Released: TBD
+
+EDB Postgres Distributed version 5.5.0 is a minor version of EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.5. See [PGD/TPA upgrades](../upgrades/tpa_overview) for details.
+!!!
+
+
+## Highlights of EDB Postgres Distributed 5.5.0
+
+Highlights of this 5.5.0 release include:
+
+* read scalability enhancements in PGD Proxy which allow read-only queries to be routed to nodes that are not the write leader, improving the overall performance of the cluster.
+
+## Compatibility
+
+!!! Note EDB Server version compatibility
+This version requires the recently released Postgres versions 14.10, 15.4,
+or 16.1 (or later) of EDB Postgres Advanced Server or EDB Postgres Extended
+Server. No such restrictions exist for PostgreSQL Server.
+
+Package managers on Debian, RHEL, and SLES pull in the required EDB Postgres
+Advanced Server or EDB Postgres Extended upgrades with an upgrade of EDB
+Postgres Distributed.
+!!!
+
+## Features
+
+| Component | Version | Description                                   | Addresses |
+|-----------|---------|-----------------------------------------------|-----------|
+| BDR       | 5.5.0   | Add support for readonly proxy routing        |           |
+| CLI       | 5.5.0   | Create PGD CLI binaries for MacOS and Windows |           |
+| Proxy     | 5.5.0   | Add support for readonly proxy routing        |           |
+
+
+## Enhancements
+
+| Component | Version | Description                                                                                                             | Addresses |
+|-----------|---------|-------------------------------------------------------------------------------------------------------------------------|-----------|
+| BDR       | 5.5.0   | Improve bulk INSERT/UPDATE/DELETE performance by sending multiple messages together in a group rather than individually |           |
+| BDR       | 5.5.0   | Log completion of an extension upgrade                                                                                  |           |
+| BDR       | 5.5.0   | Add restrictions for group commit options                                                                               |           |
+| BDR       | 5.5.0   | Ensure bdr.remove_commit_scope() handles non-existent commit scope                                                      |           |
```

Comment:
This is a bugfix rather than an enhancement (unless we count "not crashing the cluster" as an enhancement ;) ).